tests/test_run.py

```diff
@@ -72,6 +73,16 @@ def test_sac_td3(model_class) -> None:
     model.learn(110)
 
 
+@pytest.mark.parametrize("model_class", [SAC, TD3, DDPG])
+def test_sac_td3_policy_kwargs(model_class) -> None:
+    policy_kwargs = dict(activation_fn=nn.leaky_relu, net_arch=dict(pi=[64, 64], qf=[64, 64]))
```

Comment:
```suggestion
    policy_kwargs = dict(activation_fn=nn.leaky_relu, net_arch=dict(pi=[8], qf=[8]))
```

to run the tests as fast as possible